### DISCUSSION TITLE
Welcome to install.doctor Discussions!

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [megabyte-labs/install.doctor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by silopolis** [comment]
Hello,

First, thanks for sharing this huge work in the open :pray: 

After decades, one more failing hard drive and the assorted stress and burden of reinstalling/migrating my trusty and beloved workstation decided me to go a step further in handling this. As a DevOps pro, the time to apply good practices of my field to my setup is long overdue, so let's make it happen! This naturally led me to chezmoi which, in turn, from its related project page, made me discover that great project.

Debian Testing, in some form or another, has been my daily driver for nearly 15 years. She surely has her character, but she never let me down, so no reason to change. I'm using Kali to get a base KDE desktop on top of it, and then apply tens of customizations.
I also switched to using Brew to manage all my DevOpsy toolbox and environment.

For the future, I have Kaisen Linux in sight. Also Debian Testing based, but with a twist towards *admins. The project is very small and looking for contributors.

Finally, as a DevOps mentor, I would like to explore the possibility of building an Install.Doctor-based profile that would allow the students to easily setup their environment at the beginning of their training.

Cheers

PS: I'm also a technical writer wannabe and do tech translation to French so, if that can have value for the project...

**Comment by ProfessorManhattan** [comment]
Thanks for your interest. I have taken a break from this but let me know
how I can help. I was planning on using this to go with Qubes and macOS
HVMs.
